src/v/security/license.cc

```diff
@@ -16,9 +16,10 @@
 #include "json/validator.h"
 #include "utils/base64.h"
 
-#include <cryptopp/base64.h>
-#include <cryptopp/rsa.h>
-#include <cryptopp/sha.h>
```

Comment:
that looks kinda sketchy to me...were these unused?

Reply:
Thanks, yeah these should not have been removed. I've added them back now.